schema.json

```diff
@@ -66,35 +66,65 @@
                                 "type" : "string"
                             },
                             "mode" : {
-                                "enum" : ["read", "write", "read-write"]
+                                "$ref": "#/$defs/AccessMode"
                             }
                         },
                         "additionalProperties" : false,
                         "required" : ["bucket", "mode"]
                     }
                 },
                 "mongodb": {
-                    "type": "boolean"
+                    "type": "array",
+                    "items" : {
+                        "type" : "object",
+                        "properties" : {
+                            "db" : {
+                                "type" : "string"
+                            },
+                            "mode" : {
+                                "$ref": "#/$defs/AccessMode"
```

Comment:
I don't think a "write"-only option exists for MongoDB. Maybe make specific AccessModes for S3 and MongoDB.

Reply:
fixed